services/evaluator/tests/test_pytest_debug.py

```
@@ -0,0 +1,43 @@
+            "Evaluate technical skills, problem-solving, and communication on a scale of 1-10.",
+            "Candidate discussed Python algorithms effectively, implemented binary search correctly."
+        )
+        print(f"SUCCESS: Got response of length {len(result)}")
+        print(f"First 100 chars: {result[:100]}")
+        
+        # Check for expected content
+        result_lower = result.lower()
+        has_content = any(word in result_lower for word in ['technical', 'python', 'experience', 'skills', 'algorithm', 'candidate'])
+        print(f"Has expected content: {has_content}")
+        
+        return result
```

Ruff (0.13.3)

Consider moving this statement to an `else` block

(TRY300)